src/python/pants/backend/project_info/count_loc.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import Tuple
+
+from pants.core.util_rules.external_tool import (
+    DownloadedExternalTool,
+    ExternalTool,
+    ExternalToolRequest,
+)
+from pants.engine.console import Console
+from pants.engine.fs import Digest, MergeDigests, SourcesSnapshot
+from pants.engine.goal import Goal, GoalSubsystem
+from pants.engine.platform import Platform
+from pants.engine.process import Process, ProcessResult
+from pants.engine.rules import Get, collect_rules, goal_rule
+from pants.option.custom_types import shell_str
+from pants.util.enums import match
+from pants.util.logging import LogLevel
+from pants.util.strutil import pluralize
+
+
+class SuccinctCodeCounter(ExternalTool):
+    """The Succinct Code Counter (https://github.com/boyter/scc)."""
+
+    options_scope = "succinct-code-counter"
```

Comment:
I think `scc` as the scope is fine, and easier to use if we move the `--args` to here, as I suggest below.